pyccel/commands/pyccel_clean.py

```diff
@@ -12,75 +12,86 @@
 
 ext_suffix = sysconfig.get_config_var('EXT_SUFFIX')
 
-def pyccel_clean(path_dir = None, recursive = True, remove_shared_libs = False, remove_programs = False):
-    """
-    Remove folders generated by Pyccel.
 
-    Remove `__pyccel__X` and `__epyccel__X` folders as well
-    as any Python shared libraries from the directory path_dir.
-    The folders `__pyccel__X` are called `__pyccel__` unless
-    they were generated using `pytest-xdist`.
+def remove_pyccel_folders(file_name):
+    """Remove __pyccel__ or __epyccel__ folders"""
```

Comment:
Could you please expand this docstring and add the `Parameters` and `Returns` sections?